docs/src/content/docs/components/Complex/tutorial.mdx

```diff
@@ -0,0 +1,497 @@
+| `tour`       | `(step?: number)`  | `void`       | Starts the tutorial. If a step is provided, starts from that step instead of the first one.                                |
+| `exit`       | —                  | `void`       | Stops the tutorial and clears the active step.                                                                             |
+| `next`       | —                  | `void`       | Moves to the next step, if any.                                                                                            |
+| `previous`   | —                  | `void`       | Moves to the previous step, if any.                                                                                        |
+| `pause`      | —                  | `void`       | Pauses the tutorial.                                                             |
+| `resume`     | `(next?: boolean)` | `void`       | Resumes a paused tutorial. If `next` is `true`, it will advance to the next step on resume (useful for interactive steps). |
+| `changeStep` | `(step: number)`   | `void`       | Jumps directly to the provided step (1-based). Use this when the tutorial should react to external UI events.              |
+
+## Slots/Components
+
+The `Tutorial` component exposes:
+
+- [**Tutorial.Step**](#tutorialstep) - defines one step of the tutorial and marks the wrapped UI as the current target.
+- [**Custom Tooltip** (via tooltip prop)](#custom-tooltip) - lets you fully replace the default tooltip UI.
+
+### Tutorial.Step
+
+Wraps the element you want to highlight and registers it in the tutorial flow.
+
+#### Usage
+
+Simply wrap the target element with `Tutorial.Step` and provide the required `order` prop. Providing `title` and `content` will display them in the tooltip for this step.
+
+```tsx
+<Tutorial>
+    <Tutorial.Step order={1} title="Tabs" content="Use these to switch sections.">
+        <Tabs /> 
+    </Tutorial.Step>
+
+    <Tutorial.Step order={2} title="Side Panel" content="Use it for additional information." position="right" outset={-5}>
+        <SidePanel />
+    </Tutorial.Step>
+</Tutorial>
+```
+
+:::caution
+Make sure to provide unique `order` values for each step. The tutorial will follow the ascending order of these values.
+:::
+
+:::caution
+If a step is repeated, it may cause unexpected behavior.
+:::
+
+#### Props
+
+| Prop Name  | Type              | Default  | Description                                                              |
+| ---------- | ----------------- | -------- | ------------------------------------------------------------------------ |
+| `order`    | `number`          | —        | Required. Controls the order in which the step is shown.                 |
+| `title`    | `string`          | inherits | Tooltip title. If omitted, the previous step's title is reused.          |
+| `content`  | `string`          | inherits | Tooltip content. If omitted, the previous step's content is reused.      |
```

Comment:
Why the prev step's title and content are reused? Maybe render nothing if are omitted? :D